src/tools/kata-ctl/src/ops/volume_ops.rs

```diff
@@ -0,0 +1,78 @@
+use crate::args::{DirectVolSubcommand, DirectVolumeCommand};
+
+use anyhow::{anyhow, Ok, Result};
+use futures::executor;
+use hyper;
+use runtimes;
+use std::time::Duration;
+use url;
+
+const TIMEOUT: Duration = Duration::new(5, 0);
+const CONTENT_TYPE_JSON: &str = "application/json";
+
+pub fn handle_direct_volume(vol_cmd: DirectVolumeCommand) -> Result<()> {
+    let command = vol_cmd.directvol_cmd;
+    match command {
+        DirectVolSubcommand::Add(args) => {
+            runtimes::direct_volume::add(&args.volume_path, &args.mount_info)?;
```

Comment:
To handle the `add` command in `kata-ctl`, you need to write the mount info of a direct volume into a filesystem path. It seems you are trying to call the method in `runtime-rs` directly. `runtime-rs` need to get the mount info from the file under the directory, then it could be able to handle it (this functionality is to solve #5369 ).